pkg/ingester/series.go

```diff
@@ -158,7 +158,7 @@ func (s *memorySeries) head() *desc {
 	return s.chunkDescs[len(s.chunkDescs)-1]
 }
 
-func (s *memorySeries) samplesForRange(from, through model.Time) ([]model.SamplePair, error) {
+func (s *memorySeries) samplesForRange(from, through model.Time, deletedInterval []model.Interval) ([]model.SamplePair, error) {
```

Comment:
Why do we have any changes in the ingester? As far as I understand, the tombstones are loaded in the querier only.

Reply:
Like can we simplify this by only have the tombstones in the querier layer? I don't expect to see any deletes on the recent 6hrs of data and if any happens, we can filter it out in the querier. We shouldn't ideally change the proto files for this.